clients/callback_client.go

```diff
@@ -97,7 +97,7 @@ func (pcc *PeriodicCallbackClient) SendTranscodeStatus(tsm TranscodeStatusMessag
 
 	// Terminal callbacks are sent here in a sync manner
 	// Non-terminal callbacks are sent periodically, in an async manner
-	if tsm.IsTerminal() {
+	if tsm.IsTerminal() || tsm.SourcePlayback != nil {
```

Comment:
Is this just in here for a speed boost?

Reply:
Would this not mean that we start spamming callbacks for the entire transcode time (i.e because SourcePlayback continues being not nil)

Reply:
It seems to not work at all without this, I believe because of the transcoding status being sent without the source playback field (also I am still sending a `TranscodeStatusPreparingCompleted` update for the case where source playback isn't done. So this seems like the safest/best option.. there would also be no harm in some how persisting the source playback field for all subsequent statuses but not sure if it's necessary, a bit of extra complexity when I can only see an issue when we the status fails to send here, which hopefully is quite rare..?

Reply:
Maybe best to keep it simple like this for now and make an improvement if we see a lot of failures to send the status.

Reply:
👍  sounds good